lib/infrastructure/configurationManager.js

```diff
@@ -58,6 +58,7 @@ module.exports = function ConfigurationManager() {
                     hostName: self.config.repositoryCache.svn.host,
                     port: self.config.repositoryCache.svn.port || 7891,
                     protocol: self.config.repositoryCache.svn.protocol || 'svn',
+                    refreshDisabled: self.config.repositoryCache.svn.refreshDisabled || false, 
```

Comment:
Yes - this code is inside an explicit check for `(self.config.repositoryCache.svn && self.config.repositoryCache.svn.enabled)`, and for now I want the code to be as close to the original branch as possible.